### PR TITLE
[PRTL-2611] Merge qq/box download button.

### PR DESCRIPTION
**Pull request by yuanhengzhao**
## Ticket Number

e.g. PRTL-2611

## Environment to be used in testing

- [ ] `prod`
- [x] `dev-oicr`
- [ ] `qa` (which version?)

### CODE REVIEW

File: src/packages/@ncigdc/components/DownloadVisualizationButton.js [hunk 1]
```diff
@@ -56,130 +56,153 @@ type TProps = {
...
+              opacity: 0.5,
+            }),
+          }}
+          >
+          {supportsSvgToPng() ? (
           'PNG'
         ) : (
           <Tooltip
             Component={`
                   Download as PNG is currently unavaialable in your browser.
```
Comment (samrichca) [inline src/packages/@ncigdc/components/DownloadVisualizationButton.js]:
```suggestion
                  Download as PNG is currently unavailable in your browser.
```

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/boxplot.css [hunk 2]
```diff
@@ -15,13 +15,22 @@
...
 }
 
-.boxplot .box line {
-  stroke: #1784AC;
-}
-
 .boxplot .whisker path {
   stroke: #333;
 }
+.demographic-boxplot .box line {
```
Comment (samrichca) [inline src/packages/@ncigdc/modern_components/ClinicalAnalysis/boxplot.css]: are these colours stored as constants somewhere?

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js [hunk 3]
```diff
@@ -615,6 +615,7 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
       .filter(bin => variable.bins[bin].key !== variable.bins[bin].groupName)
       .length === 0
     : variable.continuousBinType === 'default';
+  console.log('variable.type.toLowerCase()', variable.type.toLowerCase());
```
Comment (samrichca) [inline src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: console log!
```suggestion

```

File: src/packages/@ncigdc/components/DownloadVisualizationButton.js [hunk 4]
```diff
@@ -56,130 +56,153 @@ type TProps = {
...
-          ...styles.row(theme),
-          ...(supportsSvgToPng()
+        </DropdownItem>
+      )}
+      {svg && (
+        <DropdownItem
+          className="test-download-png"
+          key="png"
+          onClick={() => {
+            if (!supportsSvgToPng()) return;
```
Comment (samrichca) [inline src/packages/@ncigdc/components/DownloadVisualizationButton.js]: wrap this in `if (supportsSvgToPng()) ( ... )`instead?

File: src/packages/@ncigdc/components/DownloadVisualizationButton.js [hunk 5]
```diff
@@ -56,130 +56,153 @@ type TProps = {
...
+          style={{
+            ...styles.row(theme),
+            ...(supportsSvgToPng()
             ? {}
             : {
-                opacity: 0.5,
-              }),
-        }}
-        onClick={() => {
-          if (!supportsSvgToPng()) return;
```
Comment (samrichca) [inline src/packages/@ncigdc/components/DownloadVisualizationButton.js]: wrap this in `if (supportsSvgToPng()) ( .. )` instead?

File: src/packages/@ncigdc/components/DownloadVisualizationButton.js [hunk 6]
```diff
@@ -56,130 +56,154 @@ type TProps = {
...
+              opacity: 0.5,
+            }),
+          }}
+          >
+          {supportsSvgToPng() ? (
           'PNG'
         ) : (
           <Tooltip
             Component={`
-                  Download as PNG is currently unavaialable in your browser.
```
Comment (caravinci) [inline src/packages/@ncigdc/components/DownloadVisualizationButton.js]: nice catch!

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/boxplot.css (hunk 2 above)
Comment (caravinci) [reply src/packages/@ncigdc/modern_components/ClinicalAnalysis/boxplot.css]: yes, and they'll be passed in as props somehow in another ticket's refactor

**Comment by caravinci** [comment]
LGTM 👍

**Pull request merged**